### PR TITLE
Jakarta csv2bufr

### PR DESCRIPTION
**Pull request by david-i-berry**
- Update of CSV2bufr material to use web-app. 
- Addition of BUFR CLI tools
- Some tidy up and consistency edits.

### CODE REVIEW

**Review by maaikelimper — COMMENTED** [review]
Could you please add the instruction to generate the token for "processes/wis2box" as part of the preparation for synop2bufr-section

Could you please mention to on/off toggle for WIS2 notifications and ask the participant to validate notifications where made on his/her wis2box-broker using MQTT explorer, in both synop2bufr and csv2bufr section

File: documentation/docs/practical-sessions/converting-synop-data-to-bufr.md [hunk 1]
```diff
@@ -27,14 +27,24 @@ as the relationship between the information contained in the FM-12 SYNOP reports
...
+        wis2box auth add-token --path processes/wis2box <token>
+        wis2box auth add-token --path collections/stations <token>
+        ```
+      where ``<token>`` is the value of the token. This can be left blank to automatically generate
+      a random token (recommended).
     - For practical purposes the exercises in this session use data from Romania, import the 
       station ``0-20000-0-15015`` into your station list and associate it with the topic
-      for your "Surface weather observations collection". This will be removed at the end of the session
+      for your "Surface weather observations collection". This will be removed at the end of the session.
+    - Make sure that you have MQTT explorer open and connected to your broker.
```
Comment (tomkralidis) [inline documentation/docs/practical-sessions/converting-synop-data-to-bufr.md:47]: MQTT Explorer

File: documentation/docs/practical-sessions/converting-synop-data-to-bufr.md [hunk 2]
```diff
@@ -81,36 +91,41 @@ Identify the number of weather reports in the message.
...
-is registered in the WIS2box you are ready to convert the data to BUFR.
+is registered in the wis2box you are ready to convert the data to BUFR.
 
-Open the WIS2box web app and navigate to the synop2bufr page using the left navigation drawer. 
+Open the wis2box web application and navigate to the synop2bufr page using the left navigation drawer. 
 Select the date using the date picker and copy and paste the FM-12 SYNOP message into the text 
 entry box. Assume today's date for demonstration purposes. Once the message has been copied, select 
-the appropriate topic to publish the data on, enter the "processes/wis2box" token and select submit.
-The data will now be converted to BUFR and the result returned to the web app.
+the appropriate topic to publish the data on, enter the "processes/wis2box" token and make sure the "publish to WIS2"
```
Comment (tomkralidis) [inline documentation/docs/practical-sessions/converting-synop-data-to-bufr.md:114]: `make sure the "publish to WIS" checkbox is checked`?

File: documentation/docs/practical-sessions/converting-synop-data-to-bufr.md [hunk 3]
```diff
@@ -120,13 +135,23 @@ The data will now be converted to BUFR and the result returned to the web app.
...
 
+As a final step navigate to the monitoring page from the left menu and confirm that the data have been published.
+
+<center><img alt="Image showing monitoring tab in on the left menu" src="../../assets/img/csv2bufr-monitoring.png"/></center>
+<center><caption>Screenshot showing the monitoring menu item</caption></center>
+
+<center><img alt="Image showing monitoring page and published data" src="../../assets/img/synop2bufr-monitoring.png"/></center>
+<center><caption>Screenshot showing the monitoring dashboard and the item published in this exercise</caption></center>
+
+You should also be able to see these notifications in MQTT explorer.
```
Comment (tomkralidis) [inline documentation/docs/practical-sessions/converting-synop-data-to-bufr.md:153]: MQTT Explorer.

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**